docs/framework/docker/console.md

```diff
@@ -0,0 +1,139 @@
+--
+title: Running Console applications in Docker
+description: Running Console applications in Docker
+keywords: .NET, container, Console, Applications, Docker
+author: spboyer
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net-framework-4.6
+ms.technology: vs-ide-deployment
+ms.devlang: dotnet
+ms.assetid: 85cca1d5-c9a4-4eb2-93e6-4f878de07fd7
+---
+# Running console applications in Windows containers
+
+Console applications are used for many purposes; from simple querying of a status to long running document image processing tasks. In either case, the ability to start up and scale these applications are met with limitations of hardware acquisitions, startup times or running multiple instances.
+
+Moving your console applications to use Docker and Windows containers allows for starting these applications from clean state, perform the operation and then shutdown cleanly. This topic will show the steps needed to move a console application to a Windows based container and start it using a PowerShell script.
+
+The sample console application is a simple example which takes an argument, a question in this case, and returns a random answer. This could take a `customer_id` and process their taxes, or create a thumbnail for an `image_url` argument.
+
+In addition to the answer, the `Environment.MachineName` has been added to the end of the answer to show when running the application locally, your local machine name should be returned and when running in a Windows Container; the container session id is returned.
+
+The completed example is available in the [dotnet/core-docs repository on GitHub](https://github.com/dotnet/core-docs/tree/master/samples/framework/docker/ConsoleRandomAnswerGenerator).
+
+You need to be familiar with some Docker terms before you begin working
+on moving your application to a container.
+
+> A **Docker image** is a read-only template that defines the environment
+> for a running container, including the OS, system components, and application(s).
+
+One important feature of Docker images is that images are composed from a
+base image. Each new image adds a small set of features to an existing
+image. 
+
+> A **Docker container** is a running instance of an image. 
+
+You scale an application by running the same image in many containers.
+Conceptually, this is similar to running the same application in multiple
+hosts.
+
+You can learn more about the Docker architecture by reading the 
+[Docker Overview](https://docs.docker.com/engine/understanding-docker/)
+on the Docker site. 
+
+## Getting started
```

Comment:
Missing a reference to needing to be on either Windows Server 2016 or Windows 10 Anniversary Update.

Also, the MVC topic has a screenshot of the Windows Container prompt that you could probably reuse to add some illustrations to this article. (And reinforce where it is at.)